.github/workflows/dispatch-all-publish-alpha.yml

```
@@ -0,0 +1,91 @@
+  group: ${{ github.workflow }}-${{ github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: windows-latest
+    outputs:
+      publishVersion: ${{ steps.parseVersion.outputs.publishVersion }}
+    steps:
+      - name: Parse Version
+        id: parseVersion
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)